lib/internal/modules/esm/loader.js

```diff
@@ -193,8 +177,8 @@ class Loader {
     if (resolve !== undefined)
       this._resolve = FunctionPrototypeBind(resolve, null);
     if (dynamicInstantiate !== undefined) {
-      this._dynamicInstantiate =
-        FunctionPrototypeBind(dynamicInstantiate, null);
+      process.emitWarning(
```

Comment:
this isn't needed

Reply:
Yeah, I started without it. But then the test still passed and I was super confused if anything actually happened. Having the warning did help me be confident that it actually ignored the hook.

That being said - happy to remove this last trace as well.